firebase-firestore/src/main/java/com/google/firebase/firestore/model/Document.java

```diff
@@ -29,18 +29,6 @@
   /** The key for this document */
   DocumentKey getKey();
 
-  /**
-   * Returns the version of this document if it exists or a version at which this document was
-   * guaranteed to not exist.
-   */
-  SnapshotVersion getVersion();
```

Comment:
I am not sure where to put this as a comment, but I think we should just set the "version" and "readTime" to `0` in setHasLocalMutations()

Reply:
See my other comment.